app/templates/index.html

```diff
@@ -25,71 +25,66 @@ <h1 class="panel__title venus">
         <ul class="list list--bare">
           {% for message in messages %}
           <li class="list__item mars">{{ loop.index }})
-            {{ message }}
+            {% if message.clickable %}
+              <a class="js-inpagelink" href="#census-access-code-answer">{{ message.text }}</a>
+            {% else %}
+              {{ message.text }}
+            {% endif %}
           </li>
           {% endfor %}
         </ul>
       </div>
     </div>
   </div>
 {% endif %}
-
 <div>
-<h2 class="saturn u-mt-l u-mb-m">Welcome to the Online Household Study (Alpha)</h2>
+<h2 class="saturn u-mt-l u-mb-m">Welcome to the UK-wide Online Household Study</h2>
```

Comment:
Does this need to match the entry in the [decisions log](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/666632219/Questions+Decisions+Log): "Welcome to the Office for National Statistic's UK-wide online study."?

Reply:
Good spot with the wording. There's also wording for `To begin the study, please enter your household access code and click "Start Now".`, which different to what was added in this PR. I guess we should use the wording in the decisions log for now?

Reply:
I think we hold off on changing any wording just yet as we should have a confirmed list of changes by the end of the day following a meeting earlier.

Reply:
Great stuff, thanks for letting me know 👍